template/config/index.js

```diff
@@ -7,7 +7,19 @@ module.exports = {
     index: path.resolve(__dirname, '../dist/index.html'),
     assetsRoot: path.resolve(__dirname, '../dist'),
     assetsSubDirectory: 'static',
-    assetsPublicPath: '/',
+    assetsPublicPath: '/',{{#cdn}}
+    packages: [
+      {
+        name: 'vue',
+        var: 'Vue',
+        path: 'dist/vue.runtime.min.js'
+      },
+      {
```

Comment:
this should be added only if vue-router is selected though.
I think we could do something smarter that checks if the packages `vue-router` and `vuex` are present in the package.json and build the `packages` array at runtime